### PR TITLE
Update sophos-central-source.md

### PR DESCRIPTION
**Pull request by keyur-sumo**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by jpipkin1** [comment]
@kimsauce, when I ran yarn to preview this PR, it gave me raw-loader errors:
>Module not found: Error: Can't resolve 'raw-loader' in '/Users/jpipkin/Documents/GitHub/forks/keyur-sumo/sumologic-documentation/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework'
Module not found: Error: Can't resolve 'raw-loader' in '/Users/jpipkin/Documents/GitHub/forks/keyur-sumo/sumologic-documentation/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework'
Module not found: Error: Can't resolve 'raw-loader' in '/Users/jpipkin/Documents/GitHub/forks/keyur-sumo/sumologic-documentation/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework'
Module not found: Error: Can't resolve 'raw-loader' in '/Users/jpipkin/Documents/GitHub/forks/keyur-sumo/sumologic-documentation/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework'
client (webpack 5.80.0) compiled with 4 errors

I ran `yarn add raw-loader` so I could successfully do a local build for the patch-3 branch.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sophos-central-source.md [hunk 1]
```diff
@@ -19,9 +19,6 @@ The Sophos Central Source ingests:
 :::note
 To link the endpoint data to the alert, you can map the `alert.ManagedAgent.ID` field from the alert response with the `endpointID` field from the endpoint response.
 :::
-:::note
-This Source is not available in the [Fed deployment](/docs/api/troubleshooting#Deployments-and-Sumo-Logic-Endpoints).
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sophos-central-source.md:23]: Do not delete this note, instead change it as `This Source is available in the [Fed deployment](/docs/api/troubleshooting#Deployments-and-Sumo-Logic-Endpoints).`

**Review by JV0812 — CHANGES_REQUESTED** [review]
Minor fix

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sophos-central-source.md (hunk 1 above)
Comment (keyur-sumo) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sophos-central-source.md:23]: Updated.

**Comment by keyur-sumo** [comment]
@JV0812 fix is made. can you please re-review?

**Review by JV0812 — APPROVED** [review]
LGTM

**Pull request merged**